doc/examples/other_model_types.ipynb

```diff
@@ -0,0 +1,296 @@
+    "            problem=problem, candidate_space=candidate_space\n",
+    "        )\n",
+    "        if iteration_results[TERMINATE]:\n",
+    "            break\n",
+    "        candidate_space = iteration_results[CANDIDATE_SPACE]\n",
+    "\n",
+    "    # Return the problem, which contains the calibrated models.\n",
+    "    return problem"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "93ffcec5-3143-4906-87a8-acf60c541ce4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run the model selection method.\n",
+    "problem = run_model_selection()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a93ab68f-7831-40bd-82b0-d365cca8008c",
+   "metadata": {},
+   "source": [
+    "## Analysis\n",
+    "\n",
+    "As when using a PEtab-compatible calibration tool, the results from calibration with statsmodels can also be analyzed with PEtab Select.\n",
+    "\n",
+    "Firstly, we get the best model identified during model selection. It matches the \"true\" model, with the same AIC as computed by statsmodels at the top of this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ba3a58af-8b13-4525-812a-9c7708292bc7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_model = petab_select.analyze.get_best(\n",
+    "    models=problem.state.models, criterion=problem.criterion\n",
+    ")\n",
+    "print(f\"\"\"\n",
+    "Best model information\n",
+    "----------------------\n",
+    "PEtab Select model hash: {best_model.hash}\n",
+    "Selected powers of t: {[int(parameter_id[1:]) for parameter_id, parameter_value in best_model.parameters.items() if parameter_value == ESTIMATE]}\n",
+    "Log-Likelihood: {best_model.get_criterion(criterion=Criterion.LLH)}\n",
+    "AIC: {best_model.get_criterion(criterion=Criterion.AIC)}\n",
+    "\"\"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "92857455-d8be-4bde-adc6-e6ab8de27748",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot AIC of all models that were calibrated.\n",
+    "\n",
+    "import petab_select.plot\n",
+    "\n",
+    "plot_data = petab_select.plot.PlotData(\n",
+    "    models=problem.state.models, criterion=problem.criterion\n",
+    ")\n",
+    "\n",
+    "petab_select.plot.scatter_criterion_vs_n_estimated(plot_data=plot_data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4c0a4f6f-9099-4e22-a589-80cebaf466cc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot the history of model selection iterations.\n",
+    "\n",
+    "draw_networkx_kwargs = {\n",
+    "    \"arrowstyle\": \"-|>\",\n",
+    "    \"node_shape\": \"s\",\n",
+    "    \"node_size\": 1000,\n",
+    "    \"edgecolors\": \"k\",\n",
+    "}\n",
+    "petab_select.plot.graph_iteration_layers(\n",
+    "    plot_data=plot_data, draw_networkx_kwargs=draw_networkx_kwargs\n",
+    ")"
```

Comment:
```suggestion
    ");"
```

Reply:
Adapt figure/font size so that the model IDs fit the boxes? Fine as is.

Reply:
Done now manually but the plotting library should be changed to have automatically-adjusted node sizes (see #152 )